check50/runner.py

```diff
@@ -299,8 +301,30 @@ def __init__(self, check_name, spec, checks_root, state=None):
         self.spec = spec
         self.checks_root = checks_root
         self.state = state
+        self.attribute_names = (
+            "internal.check_dir",
+            "internal.slug",
+            "internal.excepthook.outputs",
+            "internal.excepthook.output_file",
+            "internal.excepthook.verbose"
```

Comment:
Do we also need to include anything in 'internal' that would potentially be used by extensions?

Reply:
Currently in our model, extensions only come into play when the checks' module is loaded. That happens inside the `CheckRunner` and that's currently unchangeable for extensions.  So the rationale here is, all that needs to be ported over to the individual check processes is in our control, and those things just so happen to live in `internal`.  

That said, if this does grows much further than this, or extension writers do require access, this should probably change to something similar in spirit to flask's `g`. Where if something needs to persist across processes, you assign it to some namespace / global store, and then `run_check` just copies from that namespace / store. 

But for now, let's KISS

Reply:
To future proof this a bit more, let me go ahead and change `attribute_names` from an instance variable to a class variable named `CROSS_PROCESS_ATTRIBUTES`. This should make it possible for others to change this list (okay, technically tuple).